packages/cross_file/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.3+1
+
+* Fixed lastModified unimplemented error description.
```

Comment:
Fixes

See the changelog style guide linked from the PR checklist.

Reply:
Also, please add backticks around `lastModified` since it's code.

Reply:
fixed